tests/js/server/shell/shell-collection-failures-noncluster.js

```diff
@@ -254,7 +254,7 @@ function DocumentOperationsFailuresSuite() {
       let c = db._create(cn);
       c.insert({ _key: "testi", value: 1 });
 
-      internal.debugSetFailAt("RocksDBCollection::modifyFail3Always");
+      internal.debugSetFailAt("RocksDBCollection::modifyFail2Always");
```

Comment:
Why a different failure point here? I guess this is due to the renaming of the failure points in `RocksDBCollection`, but the question still is why is this needed?